server/channels/app/post_metadata_test.go

```diff
@@ -487,6 +487,54 @@ func TestPreparePostForClient(t *testing.T) {
 		})
 	})
 
+	t.Run("opengraph unsafe links", func(t *testing.T) {
+		th := setup(t)
+		defer th.TearDown()
+
+		noAccessServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+			assert.Fail(t, "acessed server")
+		}))
+
+		for _, tc := range []struct {
+			name string
+			link string
+		}{
+			{
+				name: "normal link",
+				link: noAccessServer.URL,
+			},
+			{
+				name: "normal image",
+				link: noAccessServer.URL + "/image.png",
+			},
+			{
+				name: "markdown",
+				link: "[markdown](" + noAccessServer.URL + ") link",
+			},
+			{
+				name: "markdown image",
+				link: "![markdown](" + noAccessServer.URL + "/image.png) link",
+			},
+		} {
+			t.Run(tc.name, func(t *testing.T) {
```

Comment:
Probably, I would add another `t.Run` as part of this loop, to test the other case, the one with `UnsafeLinksPostProp` not set, or set to `"false"`.

I don't think this is an stopper. but It would be a nice to have for completeness. (I know that is probably already tests in other places, but here we are specifically testing the behavior of the combination with the flag).

Reply:
Makes sense. You are kinda able to test the tests by doing that.